README.rst

```diff
@@ -2,7 +2,7 @@
   MDAnalysis Repository README
 ================================
 
-|build| |cov| [*]_
+|numfocus| |build| |cov| [*]_
```

Comment:
looks good

Reply:
So even if I select a "changes requested", it will automatically turn my review in the "comment" type. That's fair.

Reply:
@richardjgowers please ignore this whole review. It's a test but I haven't figure out how to remove it.